src/plugins/es_ui_shared/public/components/cron_editor/cron_monthly.tsx

```diff
@@ -18,13 +18,21 @@
  */
 
 import React, { Fragment } from 'react';
-import PropTypes from 'prop-types';
 import { FormattedMessage } from '@kbn/i18n/react';
 import { i18n } from '@kbn/i18n';
+import { EuiFlexGroup, EuiFlexItem, EuiFormRow, EuiSelect, EuiSelectOption } from '@elastic/eui';
 
-import { EuiFlexGroup, EuiFlexItem, EuiFormRow, EuiSelect } from '@elastic/eui';
+interface Props {
+  minute?: string;
+  minuteOptions: EuiSelectOption[];
+  hour?: string;
+  hourOptions: EuiSelectOption[];
+  date?: string;
+  dateOptions: EuiSelectOption[];
+  onChange: ({ minute, hour, date }: { minute?: string; hour?: string; date?: string }) => void;
```

Comment:
I considered reusing `FieldToValueMap` here, but decided against it. `FieldToValueMap` creates some helpful coupling between these components and `<CronEditor />` but the greater downside is that it obscures which  specific fields are actually being provided to the `onChange` callback.